From ae8a49ecc266430869f5ceff1200d5f3580d44e2 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Wed, 31 Jan 2024 16:07:34 +0100
Subject: [PATCH 1/4] data: Fix long summary after Flathub review

Rephrase app summary so that Flathub does not consider it to be too
long:
https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines/#not-too-long-1
---
 data/org.gnome.Rhythmbox3.appdata.xml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/org.gnome.Rhythmbox3.appdata.xml.in b/data/org.gnome.Rhythmbox3.appdata.xml.in
index 02bb503aa820..abc33d272928 100644
--- a/data/org.gnome.Rhythmbox3.appdata.xml.in
+++ b/data/org.gnome.Rhythmbox3.appdata.xml.in
@@ -6,7 +6,7 @@
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0+ and GFDL-1.3</project_license>
   <name>Rhythmbox</name>
-  <summary>Play and organize your music collection</summary>
+  <summary>Play and organize all your music</summary>
   <description>
     <p>
       Rhythmbox is a music management application, designed to work well under the GNOME
-- 
2.43.0


From bd942b695b5c3582effb40a11fe7d345ed5f79a7 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Wed, 31 Jan 2024 21:39:57 +0100
Subject: [PATCH 3/4] data: Fix "appstream-missing-developer-name" linter error

See https://docs.flathub.org/docs/for-app-authors/linter/#appstream-missing-developer-name
---
 data/org.gnome.Rhythmbox3.appdata.xml.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/org.gnome.Rhythmbox3.appdata.xml.in b/data/org.gnome.Rhythmbox3.appdata.xml.in
index abc33d272928..8aba484f3637 100644
--- a/data/org.gnome.Rhythmbox3.appdata.xml.in
+++ b/data/org.gnome.Rhythmbox3.appdata.xml.in
@@ -7,6 +7,7 @@
   <project_license>GPL-2.0+ and GFDL-1.3</project_license>
   <name>Rhythmbox</name>
   <summary>Play and organize all your music</summary>
+  <developer_name>The Rhythmbox developers</developer_name>
   <description>
     <p>
       Rhythmbox is a music management application, designed to work well under the GNOME
-- 
2.43.0


From 0502dbe2f9b1d8a28e5f483283be963be614f188 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Wed, 31 Jan 2024 21:41:12 +0100
Subject: [PATCH 4/4] data: Fix "appstream-screenshot-missing-caption" linter
 error

See https://docs.flathub.org/docs/for-app-authors/linter/#appstream-screenshot-missing-caption
---
 data/org.gnome.Rhythmbox3.appdata.xml.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/org.gnome.Rhythmbox3.appdata.xml.in b/data/org.gnome.Rhythmbox3.appdata.xml.in
index 8aba484f3637..b247924985b9 100644
--- a/data/org.gnome.Rhythmbox3.appdata.xml.in
+++ b/data/org.gnome.Rhythmbox3.appdata.xml.in
@@ -27,6 +27,7 @@
   <screenshots>
     <screenshot type="default">
       <image>https://gitlab.gnome.org/GNOME/rhythmbox/-/raw/master/data/screenshots/rhythmbox-main-window.png?inline=false</image>
+      <caption>Main Window</caption>
     </screenshot>
   </screenshots>
   <update_contact>jmatthew@gnome.org</update_contact>
-- 
2.43.0

